File: vm_Assignments/vm/Beginning/L2.8/sum.py
def compute_sum(n):
    sum = 0
    for i in range(1, n+1):
        sum = sum + i
    return sum


# create a table from 9 down to 1
print("N      compute_sum(N)")
print("---------------------")
for i in range(9, 0, -1):
    print(i, "         ", compute_sum(i))

File: vm_Assignments/vmc/L2.8/sum.c
#include<stdio.h>
int compute_sum(int n)
{
    int sum=0;
    for(int i=1; i<n+1; i++)
        sum += i;
    return sum;
}
int main()
{
    printf("N           compute_sum(N)\n");
    printf("--------------------------\n");
    for(int i=9; i>0; i--)
        printf("%d\t\t%d\n",i, compute_sum(i));
}
